### PR TITLE
Add Nuxt 3 example

### PR DESCRIPTION
**Pull request by wobsoriano**
Adds a Basic Nuxt example. Basically converted the Nextjs demo to Nuxt.

### CODE REVIEW

File: nuxt/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+node_modules
```
Comment (smaye81) [inline nuxt/.gitignore:1]: Can you remove this and add these to the main `.gitignore` at the project root?

File: nuxt/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "buf-nuxt",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "nuxt build",
```
Comment (smaye81) [inline nuxt/package.json:6]: Can you add a step here to the build for compiling TypeScript?

**Comment by smaye81** [comment]
Hey @wobsoriano thanks for the PR!  Left a few small comments.  Also two small overall comments:

- Please add a description of this directory to the main README.  You can follow the examples there.
- Please add a command to `update.sh` at the project root for updating the @bufbuild deps for this project.  You'll see the other examples, should be pretty self-explanatory.

**Review by smaye81 — CHANGES_REQUESTED** [review]

File: nuxt/.gitignore (hunk 1 above)
Comment (wobsoriano) [reply nuxt/.gitignore]: Removed. `node_modules` already in project root :)

File: nuxt/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare",
+    "buf:generate": "buf generate buf.build/bufbuild/eliza",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
+  },
+  "devDependencies": {
```
[the remark this answers is not in the document]
Comment (wobsoriano) [reply nuxt/package.json:15]: Updated and ran `buf:generate` again.

File: nuxt/package.json (hunk 2 above)
Comment (wobsoriano) [reply nuxt/package.json:6]: Do we have a sample of this in other examples?

Comment (smaye81) [reply nuxt/package.json:6]: Yep, there's a few.  Take a look at the [package.json](https://github.com/bufbuild/connect-web-integration/blob/main/react/vite/package.json#L8) in `react/vite` for a similar example.

Comment (smaye81) [reply nuxt/package.json:6]: @wobsoriano Just checking in.  Any update?

Comment (wobsoriano) [reply nuxt/package.json:6]: Hey, got busy. Will defo update tom 👍🏼

**Comment by 4F2E4A2E** [comment]
Can we still have a nuxt example?

**Pull request closed without merging**